src/app/modules/vault-filter/components/status-filter.component.html

```diff
@@ -3,7 +3,13 @@ <h2 class="sr-only">{{ "filters" | i18n }}</h2>
   <ul class="filter-options">
     <li class="filter-option" [ngClass]="{ active: activeFilter.status === 'all' }">
       <span class="filter-buttons">
-        <button class="filter-button" appStopClick appBlurClick (click)="applyFilter('all')">
+        <button
+          class="filter-button"
+          appStopClick
+          appBlurClick
+          (click)="applyFilter('all')"
+          appA11yTitle="{{ 'applyFilter' | i18n }}"
```

Comment:
Bit of a problem here with the a11y stuff: the `appA11yTitle` will override the text in this button, so a screenreader will just say: "apply filter" without announcing what the filter _is_.

In this case, the icon is just decorative, the actual meaning comes from the text. So you can remove `appA11yTitle` and let the text speak for itself.

You only need accessibility attributes on buttons if the icon is semantic. Sorry if I misled you here. Most of your changes around this are fine, just check where you're accidentally overriding button text.

Reply:
Thanks for that clarification. I'm glad I toned down the original plan of passing a variable into the localization string for each of these 😅

[[fix] Only use appA11yTitle for semantic buttons](https://github.com/bitwarden/desktop/pull/1487/commits/9d65f6e1033812877f4af5c669078e1ca39b2a17)